doc-theme/theme.scss

```diff
@@ -17,21 +16,32 @@ $posit-green:   #72994E;
 $posit-burgundy:#9A4665;
 $posit-burgundy-1:#78384F;
 $posit-burgundy-2:#542938;
+
+$primary: $posit-blue;
 // scss-docs-end color-variables
 
 // Typography
 // Font, line-height, and color for body text, headings, and more.
 
 //scss-font-start font-variables
-$font-family-sans-serif: "Open Sans",   system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji" !default;
-$font-size-base: 1rem !default;
+$font-family-sans-serif: "Open Sans", "Segoe UI", Roboto, "Helvetica Neue", "Noto Sans", "Liberation Sans", Arial, sans-serif, system-ui;
+$font-family-monospace: "Source Code Pro", monospace;
+$font-family-emoji: "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+$font-size-base: 1rem;
 // scss-docs-end font-variables
 
 $body-color: $posit-gray;
 
 // scss-docs-start font-sizes
 $h1-font-size:44px;
 $h2-font-size:28px;
+
+h1 {
+  size: $h1-font-size;
+}
+h2 {
+  size: $h2-font-size;
+}
```

Comment:
you can probably combine these into standard CSS rules (`h1 { size: 44px }`) unless you expect to be setting other things with these variables. Either way is fine, but you need to have them used within a rule somewhere to get picked up.

Reply:
in that case, the variables file is probably being pulled into other Sass files and used in style rules there :)